### PR TITLE
Checking the registry if theme is present and then logging it.

### PR DESCRIPTION
**Pull request by prafulVaishnav**

### CODE REVIEW

**Comment by prafulVaishnav** [comment]
@nethip Can you please look into this PR

File: src/extensions/default/HealthData/HealthDataUtils.js [hunk 1]
```diff
@@ -67,5 +70,43 @@ define(function (require, exports, module) {
...
+        if (!ExtensionManager.hasDownloadedRegistry) {
+            ExtensionManager.downloadRegistry().done(function () {
+                bracketsTheme = ExtensionManager.extensions[installedTheme];
+                if (bracketsTheme && bracketsTheme.installInfo && bracketsTheme.installInfo.locationType === ExtensionManager.LOCATION_USER && bracketsTheme.registryInfo) {
+                    result.resolve(installedTheme);
+                } else {
+                    result.reject();
+                }
+            })
+                .fail(function () {
```
Comment (nethip) [inline src/extensions/default/HealthData/HealthDataUtils.js]: nit: formatting

**Comment by nethip** [comment]
@prafulVaishnav Do we need to update our unit tests with the new changes?

**Comment by nethip** [comment]
LGTM. Please run the unit tests and let me know if they are passing, so that I can go ahead with the merge.

**Comment by prafulVaishnav** [comment]
@nethip Unit Test Cases are working fine..

**Pull request merged**